backtrace-library/src/main/cpp/CMakeLists.txt

```diff
@@ -32,9 +32,13 @@ set_property(TARGET base PROPERTY IMPORTED_LOCATION ${PROJECT_SOURCE_DIR}/crashp
 # Crashpad Headers
 include_directories(${PROJECT_SOURCE_DIR}/crashpad-builds/headers/ ${PROJECT_SOURCE_DIR}/crashpad-builds/headers/third_party/mini_chromium/mini_chromium/)
 
-# Bun Libraries
-set(LIBUNWINDSTACK_ENABLED TRUE)
-add_subdirectory(libbun)
+if(NOT ANDROID_PLATFORM MATCHES "^android-([0-9]|1[0-9]|2[0-2])$")
```

Comment:
do you think it would be clearer to do `if(ANDROID_PLATFORM MATCHES "...") message() else () ...` ?